nuget/lib/dependabot/nuget/file_updater.rb

```diff
@@ -25,7 +25,8 @@ def self.updated_files_regex
           /^dotnet-tools\.json$/i,
           /^Directory\.Build\.props$/i,
           /^Directory\.Build\.targets$/i,
-          /^Packages\.props$/i
+          /^Packages\.props$/i,
```

Comment:
I would probably revert these changes and instead identify package lock file that need updating based on the projects being updated in the native tool.